src/main/java/uk/gov/pay/directdebit/mandate/dao/mapper/GoCardlessMandateMapper.java

```diff
@@ -1,22 +1,26 @@
 package uk.gov.pay.directdebit.mandate.dao.mapper;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
 import org.jdbi.v3.core.mapper.RowMapper;
 import org.jdbi.v3.core.statement.StatementContext;
+import uk.gov.pay.directdebit.common.model.subtype.gocardless.GoCardlessCreditorId;
 import uk.gov.pay.directdebit.mandate.model.GoCardlessMandate;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
 public class GoCardlessMandateMapper implements RowMapper<GoCardlessMandate> {
     private static final String ID_COLUMN = "id";
     private static final String MANDATE_ID_COLUMN = "mandate_id";
     private static final String GOCARDLESS_MANDATE_ID_COLUMN = "gocardless_mandate_id";
+    private static final String GOCARDLESS_CREDITOR_ID = "gocardless_creditor_id";
 
     @Override
     public GoCardlessMandate map(ResultSet resultSet, StatementContext statementContext) throws SQLException {
         return new GoCardlessMandate(
                 resultSet.getLong(ID_COLUMN),
                 resultSet.getLong(MANDATE_ID_COLUMN),
                 resultSet.getString(GOCARDLESS_MANDATE_ID_COLUMN),
-                null);
+                null,
+                GoCardlessCreditorId.of(resultSet.getString(GOCARDLESS_CREDITOR_ID)));
```

Comment:
Is this safe to do? Existing GoCardless mandates in our database aren’t going to have a `gocardless_creditor_id` value, which means you’ll be passing `null` to the `GoCardlessCreditorId.of(String)`, which will go _kaboom!_ because it doesn’t accept `null` parameters.

If you look at `GatewayAccountMapper`, you can see how we work around this for other entities (tl;dr: with some horrible `if` statements).

Reply:
As discussed it is now safe because we merged https://github.com/alphagov/pay-direct-debit-connector/pull/253